### PR TITLE
Storage - Update storage.md

### PR DESCRIPTION
**Pull request by simonaco**
Create management and client sections. Add container creation code in client sample. Add management example code

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/storage.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-5/docs-ref-services/storage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C15%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707151545202088-89%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/storage.md [hunk 1]
```diff
@@ -25,39 +25,71 @@ Use the Azure Storage client module to:
...
+### Install packages with npm
 
 ```bash
 npm install azure-storage
 ```
 
-### Management
+### Example
+
+Create a storage container and upload a local file *data.txt* to it.
```
Comment (johnpapa) [inline docs-ref-services/storage.md]: file names and keywords should use ticks, like this ... 

`data.txt`

File: docs-ref-services/storage.md [hunk 2]
```diff
@@ -25,39 +25,71 @@ Use the Azure Storage client module to:
...
+const blobService = azure.createBlobService(storageConnectionString);
+
+const container = 'taskcontainer';
+const task = 'taskblob';
+const filename = 'data.txt';
+
+blobService.createContainerIfNotExists(container, error => {
+  if (error) return console.log(error);
+  blobService.createBlockBlobFromLocalFile(
+    container,
```
Comment (johnpapa) [inline docs-ref-services/storage.md]: did "prettier" make these parameters go to separate lines? seems like they should fit ok on one

File: docs-ref-services/storage.md [hunk 3]
```diff
@@ -25,39 +25,71 @@ Use the Azure Storage client module to:
...
 
 ```bash
 npm install azure-arm-storage
 ```
 
-## Example
+### Example
 
-Write a local file *data.txt* to an existing blob storage container.
+Authenticate, create client and list storage accounts.
```
Comment (johnpapa) [inline docs-ref-services/storage.md]: missing the word "the" 

> ... create the client ...

> ... list the storage accounts

File: docs-ref-services/storage.md [hunk 4]
```diff
@@ -25,39 +25,71 @@ Use the Azure Storage client module to:
...
+    );
+    return client.storageAccounts.list();
+  })
+  .then(accounts => console.dir(accounts, { depth: null, colors: true }))
+  .catch(err => console.log(err));
 ```
 
 ## Samples
 
 [!INCLUDE [node-storage-samples](../docs-ref-conceptual/includes/storage-samples.md)]
```
Comment (johnpapa) [inline docs-ref-services/storage.md]: did you verify the samples are good?

File: docs-ref-services/storage.md [hunk 5]
```diff
@@ -25,39 +25,71 @@ Use the Azure Storage client module to:
 
 Create, update, and manage Azure Storage accounts and query and regenerate access keys from your Node.js apps with the management modules.
 
-## Install modules with npm
+## Client Package
 
-Use npm to install the Azure storage client or management modules.
-
-### Client
+### Install packages with npm
```
Comment (johnpapa) [inline docs-ref-services/storage.md]: Add missing ...

> Install the azure storage client npm module

**Comment by johnpapa** [comment]
added intros for heading sections

needs review before #sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/storage.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-5/docs-ref-services/storage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180101468373-89%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/storage.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-5/docs-ref-services/storage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180423135694-89%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/storage.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-5/docs-ref-services/storage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180423376025-89%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
LGTM

#sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/storage.md](https://github.com/simonaco/azure-docs-sdk-node/blob/patch-5/docs-ref-services/storage.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707181340431729-89%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**